components/organisms/footer.html

```diff
@@ -0,0 +1,93 @@
+<div class=footer>
+
+  <footer class="main-footer">
+
+    <div class="footer-column">
+      <h4>Example</h4>
+      <button class="footer-button" type="button">Click here!</button>
+    </div>
+
+    <div class="social footer-column">
+      <h4>Connect with us!</h4>
+      <ul class="social-icons">
+        <li><a href="http://www.facebook.com"><img src='../../img/icon_facebook.png' /></a></li>
+        <li><a href="http://www.twitter.com"><img src='../../img/icon_instagram.png' /></a></li>
+        <li><a href="http://www.youtube.com"><img src='../../img/icon_twitter.png' /></a></li>
+      </ul>
+    </div>
+
+  </footer>
+
+</div>
+<style>
+
+  /* =================================
+  Base Layout Styles
+==================================== */
+  .footer {
+    padding-top: .75em;
+    background: rgba(232, 236, 241, 1);
+  }
+
+  .main-footer {
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    grid-gap: 25px;
+    padding: 2em 0;
+    background: rgba(54, 56, 57, .92);
+    text-align: left;
+    color: white;
+    font-family: arial;
+  }
+
+  .footer-column {
+    margin: 0 25px;
+  }
+
+  .social {
```

Comment:
I suggest following a convention where all classes start with the component name. Something like `.footer_social`.